packages/sanes-chrome-extension/src/extension/background/messages/index.ts

```diff
@@ -81,11 +81,14 @@ export async function sendGetPersonaMessage(): Promise<GetPersonaResponse> {
   });
 }
 
+export type GetRequestResponse = ReadonlyArray<Request>;
```

Comment:
I think this symbol can be removed. It belongs to

```
getQueuedRequests(): ReadonlyArray<Request>
```

not here and for `ReadonlyArray<Request>` or `readonly Request[]` (new notation), we probably don't need an alias